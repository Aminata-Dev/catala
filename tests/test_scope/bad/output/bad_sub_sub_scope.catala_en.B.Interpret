[ERROR] There is a conflict between multiple validd consequences for assigning the same variable.

This consequence has a valid justification:
  --> test_scope/bad/bad_sub_sub_scope.catala_en
   | 
25 |   definition y under condition a2.x + 1 = 2 consequence equals 1
   |                                                                ^
   + Article

This consequence has a valid justification:
  --> test_scope/bad/bad_sub_sub_scope.catala_en
   | 
24 |   definition y under condition a2.x = 1 consequence equals 1
   |                                                            ^
   + Article
