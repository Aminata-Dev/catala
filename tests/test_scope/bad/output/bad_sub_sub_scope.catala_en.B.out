[ERROR] There is a conflict between multiple exceptions for assigning the same variable.

This justification is true:
  --> test_scope/bad/bad_sub_sub_scope.catala_en
   | 
25 |   definition y under condition a2.x + 1 = 2 consequence equals 1
   |                                ^^^^^^^^^^^^
   + Article

This justification is true:
  --> test_scope/bad/bad_sub_sub_scope.catala_en
   | 
24 |   definition y under condition a2.x = 1 consequence equals 1
   |                                ^^^^^^^^
   + Article
